README.md

```diff
@@ -162,7 +162,32 @@ const lc = new LockContext()
 ```
 
 > [!NOTE]
-> At the time of this writing, the default LockContext is set to use the `sub` Identity Claim, as this is the only Identity Claim that is currently supported.
+> At the time of this writing, the default LockContext context is set to use the `sub` Identity Claim.
```

Comment:
That's only if you provide a `LockContext` though right? The sentence reads like there will _always_ be a lock context and if you don't specify anything different it will be `sub`. Maybe make it clear that there is no lock context by default but if provided it will be set to `sub` by default?